src/client/common.ts

```diff
@@ -89,17 +89,17 @@ function setFlattenedQueryParams(urlSearchParams: URLSearchParams, parameter: an
     if (typeof parameter === "object") {
         if (Array.isArray(parameter)) {
             (parameter as any[]).forEach(item => setFlattenedQueryParams(urlSearchParams, item, key));
-        } 
```

Comment:
@xinyu-li-cb why did we change this file?

Reply:
I just copied over all the generated files, likely something wrong with my generator, let me remove it.

Reply:
my ide will remove surfix space...